src/states/elder/mod.rs

```diff
@@ -714,7 +714,20 @@ impl Elder {
             msg.routing_message()
         );
 
-        if self.in_location(msg.message_dst()) {
+        let in_location = self.in_location(msg.message_dst());
+        if !in_location || msg.message_dst().is_multiple() {
+            // Relay closer to the destination or broadcast to the rest of our section.
+            if let Err(error) = self.send_signed_message(&msg) {
+                debug!(
+                    "{} Failed to send {:?}: {:?}",
+                    self,
+                    msg.signed_routing_message(),
+                    error
+                );
+            }
+        }
+
+        if in_location {
             let signed_msg = msg.signed_routing_message();
             let handle = self.verify_signed_message(signed_msg).map_err(|error| {
                 self.log_verify_failure(signed_msg, &error);
```

Comment:
Why not put the log_verify_failure from within so we can simplify the caller?

Reply:
Ha, I see what you mean by https://github.com/maidsafe/routing/pull/2018/files/453a07da18a1bf103260ccd0bea47a90d123ce21..dd0d5de157935bbc246104c7616ef2a4329559f5#r368989410